images/scripts/lib/atomic.setup

```diff
@@ -35,21 +35,10 @@ fi
 /var/lib/testvm/docker-images.setup
 
 # Download the libssh RPM plus dependencies which we'll use for
-# package overlay.  The only way to do this is via a container
-. /etc/os-release
-REPO="updates"
-if [ "$ID" = "rhel" ]; then
-    subscription-manager repos --enable rhel-7-server-extras-rpms
-    REPO="rhel-7-server-extras-rpms"
-    ID="rhel7"
-fi
-docker run --rm --volume=/etc/yum.repos.d:/etc/yum.repos.d:z --volume=/root/rpms:/tmp/rpms:rw,z "$ID:$VERSION_ID" /bin/sh -cex "yum install -y findutils createrepo yum-utils && (cd /tmp/; yumdownloader --enablerepo=$REPO libssh) && find /tmp -name '*.$(uname -m).*rpm' | while read rpm; do mv -v \$rpm /tmp/rpms; done; createrepo /tmp/rpms"
-rm -f /etc/yum.repos.d/*
-cat >/etc/yum.repos.d/deps.repo <<EOF
-[deps]
-baseurl=file:///root/rpms
-enabled=1
-EOF
+# package overlay. Use the EPEL package on RHEL Atomic as well, so that we
+# don't need a subscription.
+mkdir -p /root/rpms
+(cd /root/rpms && curl --silent --show-error -O https://dl.fedoraproject.org/pub/epel/7Server/x86_64/Packages/l/libssh-0.7.1-0.7.el7.x86_64.rpm)
```

Comment:
so we are gonna be stuck with this one version? And also wouldn't this disappear when new version is released? Koji should be safer https://koji.fedoraproject.org/koji/buildinfo?buildID=1240987

Reply:
I'm actually counting on this to fail when there ever is a new RHEL Atomic update (it seems it's not going to be support for that long any more). If there ever is, we need to update the version there -- we should at least keep up with what's in EPEL.